### PR TITLE
Nearmare Weapons Update.xml

### PR DESCRIPTION
**Pull request by 450570206**
Make the new weapons be compatible with CE

## Additions

Add the new weapons.

## Changes

Update the patch

## References

None

## Reasoning

The patch should be updated.

## Alternatives

I will try to update other patches.

## Testing

Not find any bugs.

### CODE REVIEW

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml [hunk 1]
```diff
@@ -325,10 +325,88 @@
...
+					<range>52</range>
+					<burstShotCount>10</burstShotCount>
+					<ticksBetweenBurstShots>2</ticksBetweenBurstShots>
+					<soundCast>Shot_AssaultRifle</soundCast>
+					<soundCastTail>GunTail_Medium</soundCastTail>
+					<muzzleFlashScale>11</muzzleFlashScale>
+				</Properties>
+				
+				<AmmoUser>
+					<magazineSize>45</magazineSize>
```
Comment (N7Huntsman) [inline Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:395]: 12.7x55mm is a quite large round for an assault rifle, so with a normal (non-drum) magazine, the gun can't hold quite so many. I'd reduce this to 20.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml [hunk 2]
```diff
@@ -325,10 +325,88 @@
...
+					<reloadTime>10</reloadTime>
```
Comment (N7Huntsman) [inline Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:396]: This is a very long reload for a magazine fed assault rifle. For comparison, the vanilla Assault rifle reloads in 4.6. If you wanted this to take a bit longer to reload, I'd set the time to around 5.2.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml [hunk 3]
```diff
@@ -325,10 +325,88 @@
...
+					<ammoSet>AmmoSet_127x55mmAR</ammoSet>
+				</AmmoUser>
+				
+				<FireModes>
+					<aimedBurstShotCount>5</aimedBurstShotCount>
```
Comment (N7Huntsman) [inline Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:401]: I'd reduce this to 2.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml [hunk 4]
```diff
@@ -325,10 +325,88 @@
...
+				</statBases>
+				
+				<Properties>
+					<recoilAmount>3</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_127x55mmAR_FMJ</defaultProjectile>
+					<warmupTime>5</warmupTime>
+					<range>52</range>
+					<burstShotCount>10</burstShotCount>
```
Comment (N7Huntsman) [inline Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:387]: I'd reduce this to 4. With as much recoil as this rifle has, shooters most likely won't be able to hit much of anything.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
An unusual but acceptable choice of caliber, though a few stats should be amended.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 1 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:395]: Ok. I will change it.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 2 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:396]: Ok. I will change it.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 3 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:401]: According to the author , this gun has high firing rate .So I think 5 is suitable.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 4 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:387]: According to the author , this gun has high warm up time. I want to keep a balance with cost and strength.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 1 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:395]: But I think 30 has a better balance with cost and strength.

File: Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml (hunk 4 above)
Comment (450570206) [reply Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml:387]: According to the author , this gun has high firing rate , high lethality ,low hit rate and high warm up time.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**